packages/flutter_test/test/bindings_test.dart

```diff
@@ -17,7 +17,8 @@ import 'package:flutter_test/flutter_test.dart';
 import 'package:test_api/test_api.dart' as test_package;
 
 void main() {
```

Comment:
Nit: leave the existing formatting for unrealted code

Reply:
Done